tests/storage/merge_test.py

```diff
@@ -455,8 +460,12 @@ def test_reduce_chunked(self):
                 ('reduce', (base_vol.optimal_size(),), {}),
             ]
 
-    def test_reduce_not_chunked(self):
-        with self.make_env(sd_type='file', format='cow', chain_len=4) as env:
+    @pytest.mark.parametrize("sd_type, format", [
+        pytest.param('file', 'cow'),  # Not chunked, reduce not called
+        pytest.param('block', 'raw'),  # Preallocated, reduce not called
+    ])
```

Comment:
This does not test the interesting cases - the issue is with preallocated qcow2 on block, and this tests preallocated raw.